Prevent mk from ignoring errors

By default, the plan9port INSTALL script doesn't report build error,
which can result in a mostly broken build begin installed.
--- a/INSTALL
+++ b/INSTALL
@@ -65,7 +65,6 @@
 	;;
 esac
 
-(
 if [ `uname` = SunOS ]; then
 	# On Solaris x86, uname -p cannot be trusted.
 	echo "* Running on Solaris: checking architecture..."
@@ -159,7 +158,7 @@
 		echo "* Warning: not all binaries built successfully."
 	fi
 	echo "* Installing everything in $PLAN9/bin..."
-	mk -k install || exit 1
+	mk install || exit 1
 	if [ ! -x $PLAN9/bin/cleanname -o ! -x $PLAN9/bin/acme -o ! -x $PLAN9/bin/sam ]; then
 		echo " "
 		echo "* Warning: not all binaries built successfully."
@@ -211,5 +210,4 @@
 	echo "	PLAN9=$PLAN9 export PLAN9"
 	echo '	PATH=$PATH:$PLAN9/bin export PATH'
 fi
-) 2>&1 | tee install.log | $awk -f $PLAN9/dist/isum.awk -v 'copy='install.sum
 
